hr_timesheet_sheet/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2018 Eficent
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'HR Timesheet Sheet',
+    'version': '11.0.1.0.0',
+    'category': 'Human Resources',
+    'sequence': 80,
+    'summary': 'Timesheet Sheets, Activities',
+    'license': 'AGPL-3',
+    'author': 'Eficent, Odoo Community Association (OCA)',
+    'website': 'https://github.com/OCA/hr-timesheet',
+    'depends': [
+        'hr_timesheet', 'account',
```

Comment:
because v10 <code>hr_timesheet</code> has a dependency on account and v11 <code>hr_timesheet</code> doesn't have it, and I thought that the previous dependencies of <code>hr_timesheet_sheet</code> (although indirectly) should remain.

Reply:
Do you think I should remove it?

Reply:
I think so. I don't think you made use of any particular field introduced by 'account'. It should be fine to remove it, please do it and I will try with some extra tests ;)